src/components/breadcrumbs/breadcrumbs.js

```diff
@@ -25,25 +30,82 @@ const limitBreadcrumbs = (breadcrumbs, max) => {
 
     if (isEven) {
       breadcrumbsAtEnd.unshift(breadcrumb);
+      breadcrumbsToRemove.push(Number(breadcrumb.key));
     } else {
       breadcrumbsAtStart.push(breadcrumb);
+      breadcrumbsToRemove.push(Number(breadcrumb.key));
     }
   }
 
+  // Sort the indices from low to high
+  breadcrumbsToRemove.sort(function(a, b) {
+    return a - b;
+  });
+
+  // Work backwards and remove each index from the overflow
+  // array so that all we have left are the items that aren't
+  // visible on screen. Only do it when showMaxPopover is true
+  if (showMaxPopover) {
+    for (let i = breadcrumbsToRemove.length - 1; i >= 0; i--) {
+      overflowBreadcrumbs.splice(breadcrumbsToRemove[i], 1);
+    }
+  }
+
+  const EuiBreadcrumbCollapsed = () => {
+    const [isPopoverOpen, setIsPopoverOpen] = useState(false);
+
+    const ellipsisButton = (
+      <EuiI18n
+        token="euiBreadcrumbs.collapsedBadge.ariaLabel"
+        default="Show all breadcrumbs">
+        {ariaLabel => (
+          <EuiBadge
+            aria-label={ariaLabel}
+            onClick={() => setIsPopoverOpen(!isPopoverOpen)}
+            onClickAriaLabel={ariaLabel}
+            title="View hidden breadcrumbs"
+            className="euiBreadcrumb euiBreadcrumbBadge">
```

Comment:
Nit that I didn't notice before, this element is a child of the euiBreadcrumb so the classname should be formatted with underscores and probably made more specific.
```suggestion
            className="euiBreadcrumb euiBreadcrumb__collapsedBadge">
```